koan/Koan/Parser.hs

```diff
@@ -0,0 +1,73 @@
+module Koan.Parser where
```

Comment:
It would probably be worth giving some concrete example of something that could be parsed by this.